src/index.ts

```diff
@@ -16,12 +16,12 @@ export = class StrongConfig {
     this.params = params
   }
 
-  public load(): ReturnType<typeof load> {
+  public load(configDir?: string): ReturnType<typeof load> {
     if (this.config) {
       return this.config
     }
 
-    this.config = load()
+    this.config = load(configDir)
```

Comment:
Can this lead to a bug when `configDir` is falsy and we're passing e.g. `null` down to the `load()` function?

Reply:
Good point, think we should check for that inside `load()`. Values of `configDir` we want to fail on are `typeof !== 'string'` which also catches `null`. Maybe we can make use of a util `isPath(configDir)` or smth too here.

Reply:
Maybe safer to also preset it to `(configDir = "./config")` in the `StrongConfig.load()` class method?

Reply:
the default param will only overwrite (not passed) or `undefined`. `null`, any `number`, etc., would all slip through

Reply:
added `path.normalize` now, which throws when a non-string is passed https://nodejs.org/api/path.html#path_path_normalize_path

Regarding adding a second default parameter:

1. I think it's redundant since we're doing that in `load()` already.
2. I'd leave all things validation/defaulting/etc inside the `load()` function. It keeps `index.ts` simple.

Reply:
perfect, sounds good